roadie-rails.gemspec

```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "roadie", "~> 3.1"
-  spec.add_dependency "railties", ">= 3.0", "< 5.1"
+  spec.add_dependency "railties", ">= 3.0", "<= 5.1"
```

Comment:
I think checking that it's less than 5.2 is better, but only because I believe that 5.1.1 wouldn't be supported by this specifier. I might be very wrong about that one. It's been a while since I did Ruby work like this.